bcos-scheduler/src/BlockExecutive.h

```diff
@@ -202,6 +204,7 @@ class BlockExecutive : public std::enable_shared_from_this<BlockExecutive>
     bcos::txpool::TxPoolInterface::Ptr m_txPool;
 
     size_t m_gasLimit = TRANSACTION_GAS;
+    std::string m_gasPrice = std::string("");
```

Comment:
Default values?

Reply:
Yes, I will change it to std::string("0x0")